File: Programming Basics with JS/T03 - Conditional Statements/exercise/08. Lunch Break.js
function brake(input){
    let nameOfMovie = (input[0]);
    let episodeDuration = Number(input[1]);
    let brakeDuration = Number(input[2]);
    let timeForRest = brakeDuration / 4;
    let timeForLunch = brakeDuration / 8;
    let totalReservedTime = timeForLunch + timeForRest;
    let freeTimeOfBrake = brakeDuration - totalReservedTime;

    if (episodeDuration <= freeTimeOfBrake) {
        let diff = Math.abs((episodeDuration + totalReservedTime) - brakeDuration);
        console.log(`You have enough time to watch ${nameOfMovie} and left with ${Math.ceil(diff)} minutes free time.`);
    } else {
        let diff = Math.abs(brakeDuration - (episodeDuration + totalReservedTime));
        console.log(`You don't have enough time to watch ${nameOfMovie}, you need ${Math.ceil(diff)} more minutes.`)
    }
}
brake(["Game of Thrones", "65", "96"])

File: Programming Basics with Python/T03 - Conditional Statements/exercise/08. Lunch Break.py
from math import ceil

series_name = input()
series_time = int(input())
lunch_break = int(input())

lunch_time = lunch_break / 8
break_time = lunch_break / 4

time_left = lunch_break - lunch_time - break_time
total_time = abs(time_left - series_time)

if time_left >= series_time:
    print(f'You have enough time to watch {series_name} and left with {ceil(total_time)} minutes free time.')
else:
    print(f"You don't have enough time to watch {series_name}, you need {ceil(total_time)} more minutes.")